src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -441,9 +441,10 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
     case Event(message: CallStartMessage, data) if !data.isPending(message.callKey) =>
       val updatedData = message.handleStatusPersist(this, data)
       val callKey = message.callKey
-
       fetchLocallyQualifiedInputs(callKey) match {
-        case Success(callInputs) => startActor(callKey, callInputs, message.startMode)
+        case Success(callInputs) =>
+          globalDataAccess.updateCallInputs(workflow.id, callKey, callInputs)
```

Comment:
If you'd been awake at 4am this morning, you'd have seen me messing around with onSuccess and onFailure here. Looks like my tidying-up has left them a bit disconnected :(